### PR TITLE
Fix MXNet backend operator bugs. Enabled Keras backend tests

### PR DESCRIPTION
**Pull request by sandeep-krishnamurthy**
Fixed bugs across operators in MXNet backend. Enabled Keras backend test.

Few operators are not supported in MXNet backend and corresponding tests are skipped for MXNet backend. I will create an issue in this repo to track them before merging this code.

With this, we will enable nightly tests for MXNet backend with Keras backend test.

@jiajiechen

### CODE REVIEW

**Comment by sandeep-krishnamurthy** [comment]
Created issue here to track the skipped or failing operators
https://github.com/deep-learning-tools/keras/issues/5

File: tests/keras/backend/backend_test.py [hunk 1]
```diff
@@ -39,9 +39,12 @@
     KMX = None
     warnings.warn('Could not import the MXNet backend')
 
+BACKENDS = set(BACKENDS)
+BACKENDS_WITHOUT_MXNET = BACKENDS - set([KMX])
```
Comment (jiajiechen) [inline tests/keras/backend/backend_test.py]: So these tests does not include MXNet backend?

**Comment by jiajiechen** [comment]
Hi Sandeep, 

Overall this looks good to me. Just curious that you changed the BACKENDS to BACKENDS_WITHOUT_MXNET, so does this mean the actual tests that are running do not cover MXNET backend? 

Or, am I missing something here?

File: tests/keras/backend/backend_test.py (hunk 1 above)
Comment (sandeep-krishnamurthy) [reply tests/keras/backend/backend_test.py]: That is correct. Tests using BACKENDS_WITHOUT_MXNET will not run for MXNet.

**Pull request merged**